packages/trader/src/Stores/Modules/Trading/Helpers/__tests__/proposal.js

```diff
@@ -19,77 +19,38 @@ describe('Proposal', () => {
-                },
-                growth_rate: undefined,
-                spot: undefined,
-                spot_time: undefined,
-            });
-        });
-        it('should return profit and return calculated if proposal has no error', () => {
-            const response = {
-                proposal: {
-                    proposal: 1,
-                    ask_price: 50,
-                    display_value: 200,
-                    payout: 300,
-                    id: 'id1',
-                    longcode: 'This is a longcode',
-                },
-            };
-            const obj_prev_contract_basis = {
-                text: 'payout',
-                value: 1234,
-            };
-            expect(Proposal.getProposalInfo(store, response, obj_prev_contract_basis)).to.deep.eql({
-                cancellation: undefined,
-                commission: undefined,
-                error_code: undefined,
-                error_field: undefined,
-                limit_order: undefined,
-                profit: '250.00',
-                returns: '500.00%',
-                stake: 200,
-                payout: 300,
-                id: 'id1',
-                message: 'This is a longcode',
-                has_error: false,
-                has_error_details: false,
-                has_increased: false,
-                obj_contract_basis: {
-                    text: 'Stake',
-                    value: 200,
-                },
-                growth_rate: undefined,
-                spot: undefined,
-                spot_time: undefined,
-            });
+            // const obj_prev_contract_basis = {
+            //     text: 'Payout',
+            //     value: 1234,
+            // };
+            // const response = {
+            //     error: {
+            //         message: 'This is error',
+            //     },
+            // };
+            // expect(Proposal.getProposalInfo(store, response, obj_prev_contract_basis)).toEqual({
+            //     profit: '0.00',
+            //     returns: '0.00%',
+            //     stake: undefined,
+            //     payout: undefined,
+            //     cancellation: undefined,
+            //     commission: undefined,
+            //     error_code: undefined,
+            //     error_field: undefined,
+            //     limit_order: undefined,
+            //     id: '',
+            //     message: 'This is error',
+            //     has_error: true,
+            //     has_error_details: false,
+            //     has_increased: false,
+            //     obj_contract_basis: {
+            //         text: 'Stake',
+            //         value: '',
+            //     },
+            //     growth_rate: undefined,
+            //     spot: undefined,
+            //     spot_time: undefined,
+            // });
         });
         // it('should return profit and return calculated if proposal has no error', () => {
         //     const response = {
```

Comment:
@maryia-deriv, Are you guys planning to use this file later? It seems like you're not using it why not get rid of it?

Reply:
@jim-deriv Yes, we need to make changes to this test file. It's commented out on master because @niloofar-deriv was doing some work on the tests as I remember. When/If she uncomments it, she will be able to use it with our changes applied. Without our changes, these tests will fail.

Reply:
@maryia-deriv I finished my changes on these test files and if you pull the latest master || develop you will get the refactored tests

Reply:
@niloofar-deriv unfortunately, i can pull only from master, and i have the latest from there already but i guess that when your change on develop appears on master later, i'll see it and will have to resolve the conflict by adding additional fields related to Accumulators to this test, otherwise it will fail. don't worry guys😊

Reply:
Oh I see, so let's wait till they merge develop with master :grin: